### PR TITLE
Code Readability - Gamemodes

### PR DESCRIPTION
**Pull request by KalevTait**
## What Does This PR Do
Converts 0/1 to FALSE/TRUE where those #defines make the code more readable. Also converts one TRUE to FLASH_PROTECTION_FLASH as that is more correct in the context. Note: There are a few cases where this converts 0.0/1.0 to FALSE/TRUE - this change is consistent with how the variable are used.

## Why It's Good For The Game
More readable code is easier to work with.

### CODE REVIEW

**Comment by Sirryan2002** [comment]
Looks like all the conversion from 0 -> FALSE and 1 -> TRUE  are good.

File: code/game/gamemodes/miniantags/slaughter/slaughter.dm [hunk 1]
```diff
@@ -29,7 +29,7 @@
 	maxHealth = 200
 	health = 200
 	environment_smash = 1
-	//universal_understand = 1
+	//universal_understand = TRUE
```
Comment (SteelSlayer) [inline code/game/gamemodes/miniantags/slaughter/slaughter.dm:32]: Can you remove this commented out line instead?

Comment (KalevTait) [reply code/game/gamemodes/miniantags/slaughter/slaughter.dm]: Done

**Comment by SteelSlayer** [comment]
PR Needs to be deconflicted, then it can be merged.

**Comment by farie82** [comment]
Gonna post it on here as well so it is not just on discord.
Be sure to merge master onto this branch instead of deleting the conflicts yourself.  I've `@`ed you on discord with a guide on how to do this. If you need help do let us know

**Comment by KalevTait** [comment]
@farie82 I ended up not following your guide exactly - figured out how to merge using the command line in VS Code rather than getting GitHub desktop working. Please let me know if I've not done this correctly.

**Review by farie82 — APPROVED** [review]
LGTM. You merged master correctly into your branch

**Pull request merged**